pkg/service/credential/model.go

```diff
@@ -45,7 +47,7 @@ type GetCredentialBySchemaRequest struct {
 }
 
 type GetCredentialsResponse struct {
-	Credentials []credsdk.VerifiableCredential
```

Comment:
If this credential.CredentialsContainer becomes confusing or problematic in the future we could go another route. Basically in the SDK like you can inittilize the oebject with a JWT or a normal json parse, and it would be a []credsdk.VerifiableCredential (but with some jwt handling extra fields if it was a jwt?)

Reply:
let me take this suggestion